front/lib/actions/helpers.ts

```diff
@@ -81,18 +77,18 @@ export async function callAction<V extends t.Mixed>({
   const responseChecker = (response: unknown): response is responseType =>
     isRight(responseSchema.decode(response));
 
-  if (responseChecker(response.value)) {
+  if (responseChecker(r.value)) {
     // the response is a valid success response for the action
     // return the "value" field of the first result
-    return new Ok(response.value.results[0][0].value);
+    return new Ok(r.value.results[0][0].value);
   }
 
-  if (isActionResponseBase(response.value)) {
+  if (isActionResponseBase(r.value)) {
     // the response is of the right shape, but it's not a success response
     return new Err({
       type: "action_failed",
       message: `Doc Tracker action failed response: ${JSON.stringify(
```

Comment:
@fontanierh weird to see mentions of doc tracker here?

Reply:
Indeed, will update to remove mentions of doc tracker 👍